homework/index.js

```diff
@@ -30,13 +30,188 @@
     return elem;
   }
 
+  const root = document.getElementById('root');
+  const container = createAndAppend('div', root, { id: 'container' });
+  function getSelectedData(value, data) {
+    let selected;
+    data.forEach(item => {
+      if (item.name === value) {
+        selected = item;
+      }
+    });
+    return selected;
+  }
+
+  function generateSelections(repos) {
+    const header = createAndAppend('header', root, {});
+    const selectionElem = createAndAppend('select', header, {
+      id: 'repositories',
+    });
+    let id = 0;
+
+    repos.forEach(repo => {
+      createAndAppend('option', selectionElem, {
+        id: id++,
+        value: repo,
+        text: repo,
+      });
+    });
+  }
+
+  function generateInfoSection(selected, data) {
+    const div = createAndAppend('div', container, { id: 'infoDiv' });
+    const selectedData = getSelectedData(selected, data);
+    createAndAppend('a', div, {
+      id: 'repoName',
+      text: `Repository: ${selectedData.name}`,
+      href: selectedData.html_url,
+      target: '_blank',
+    });
+    createAndAppend('p', div, {
+      id: 'desc',
+      text: `Description: ${selectedData.description}`,
+    });
+    createAndAppend('p', div, {
+      id: 'forks',
+      text: `Forks: ${selectedData.forks}`,
+    });
+    createAndAppend('p', div, {
+      id: 'updated_at',
+      text: `Updated: ${selectedData.updated_at}`,
+    });
+  }
+
+  function generateContSection(selected) {
```

Comment:
I would suggest typing out `contributor`. Right now I'm like: "Hmmm is this 'content'? Or 'continue'? 🤔"